src/collection-browser.ts

```diff
@@ -1786,7 +1786,7 @@ export class CollectionBrowser
         dateReviewed: result.reviewdate?.value,
         description: result.description?.values.join('\n'),
         favCount: result.num_favorites?.value ?? 0,
-        href: result.__href__?.value,
+        href: this.collapseRepeatedQuotes(result.__href__?.value),
```

Comment:
q: does this need the petabox fix to go with it?

Reply:
No, the two fixes should be independent. This one only impacts the search page, while the petabox fix only impacts the (non-Offshoot) collection pages.